### PR TITLE
NEWRELIC-5414 Updated the iOS agent upgrade docs

### PR DESCRIPTION
**Pull request by mbruin-NR**
Updated the iOS agent upgrade docs to include SPM and Cocoa Pods
https://issues.newrelic.com/browse/NEWRELIC-5414

## Give us some context

The docs for updating the iOS agent didn't include instructions for SPM or cocoa pods, also the manual instructions weren't complete. I added the SPM and Cocoa Pods instructions and fixed the manual instructions.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 1]
```diff
@@ -18,17 +18,35 @@ redirects:
...
+ For information about the latest version, refer to the [release notes](/docs/releases/ios).
 
-You must be an account Admin to install, configure, and upgrade the iOS agent. For information about the latest version, refer to the [release notes](/docs/releases/ios).
+## Update with Swift Package Manager [#SPM]
 
-## Replace your iOS framework [#upgrading]
+Updating the New Relic iOS agent using Swift Package Manager:
 
-First, you'll need to replace the earlier version of your iOS agent framework:
+1. Navigate to the NewRelic agent in the Package Dependencies section of the xCode project navigator.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:27]:
```suggestion
1. Navigate to the `NewRelic agent` in the Package Dependencies section of your xCode project navigator.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 2]
```diff
@@ -18,17 +18,35 @@ redirects:
...
-1. From the Project Navigator (**CMD 1**) in Xcode, search for **NewRelic.xcframework**.
-2. Right-click or control-click **NewRelic.xcframework**, and select **Show in Finder**.
-3. Drag **NewRelic.xcframework** to the trash.
-4. Verify that the Xcode project highlights the reference to **NewRelic.xcframework** in red.
-5. Right-click or control-click **NewRelic.xcframework**, and select **Delete** to remove the obsolete reference from the project.
 
-Then, continue with the installation process in the UI at **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+## Update with Cocoa Pods [#Cocoa-Pods]
+
+Updating the New Relic iOS agent using Cocoa Pods:
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:34]:
```suggestion
To update the New Relic iOS agent using Cocoa Pods:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 3]
```diff
@@ -18,17 +18,35 @@ redirects:
...
+1. Open the **Terminal** app and navigate to the directory with your PodFile.
+2. Run the following command:
+```
+pod update NewRelicAgent
+```
+
+
+## Manually replace your iOS framework [#Manual]
+
+Downloading and manually installing the New Relic iOS agent.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:45]:
```suggestion
To download and manually install the New Relic iOS agent:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 4]
```diff
@@ -18,17 +18,35 @@ redirects:
...
+
+1. First click this [link](https://download.newrelic.com/ios_agent/) to be taken to our iOS agent hosting site.
+2. Click the version of the iOS agent you want to update to. This will begin the download.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:48]:
```suggestion
```suggestion
1. Go to https://download.newrelic.com/ios_agent/ and select the iOS agent you want to update to. A download will be initiated. 
```
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 5]
```diff
@@ -18,17 +18,35 @@ redirects:
...
+3. Navigate to where the iOS agent was downloaded and unzip it.
+4. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
+5. Right-click or control-click NewRelic.xcframework, and select Show in Finder.
+6. Drag the newly downloaded iOS agent into the same folder as the old, and select to replace the old.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:52]:
```suggestion
2. Navigate to where the iOS agent was downloaded and unzip it.
3. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
4. Right-click or control-click NewRelic.xcframework, and select **Show in Finder**.
5. Drag the newly downloaded iOS agent into the same folder as the old agent, and select to replace the old.
```

**Review by ghost — CHANGES_REQUESTED** [review]
Take a look at my suggestions.

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 6]
```diff
@@ -18,17 +18,35 @@ redirects:
...
+pod update NewRelicAgent
+```
+
+
+## Manually replace your iOS framework [#Manual]
+
+To download and manually install the New Relic iOS agent:
+
+```suggestion
+1. Go to https://download.newrelic.com/ios_agent/ and select the iOS agent you want to update to. A download will be initiated.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:48]:
```suggestion
1. Go to https://download.newrelic.com/ios_agent/ and download the iOS agent you want to update to.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 7]
```diff
@@ -18,17 +18,35 @@ redirects:
...
+```
+pod update NewRelicAgent
+```
+
+
+## Manually replace your iOS framework [#Manual]
+
+To download and manually install the New Relic iOS agent:
+
+```suggestion
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:47]:
```suggestion
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 8]
```diff
@@ -18,17 +18,34 @@ redirects:
...
+## Update with Cocoa Pods [#Cocoa-Pods]
+
+To update the New Relic iOS agent using Cocoa Pods:
+
+1. Open the **Terminal** app and navigate to the directory with your PodFile.
+2. Run the following command:
+```
+pod update NewRelicAgent
+```
+
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:41]:
```suggestion
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 9]
```diff
@@ -18,17 +18,33 @@ redirects:
...
+## Manually replace your iOS framework [#Manual]
+
+To download and manually install the New Relic iOS agent:
+
+1. Go to https://download.newrelic.com/ios_agent/ and download the iOS agent you want to update to.
+2. Navigate to where the iOS agent was downloaded and unzip it.
+3. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:48]:
```suggestion
3. From the Project Navigator (CMD 1) in Xcode, search for NewRelic.xcframework.
```
@mbruin-NR What is CMD 1? Are we suggesting a keyboard shortcut to get to the project navigator, or searching for NewRelic.xcframework?

Comment (mbruin-NR) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:48]: Yes the key command (CMD 1) will open the project navigator in xCode.

Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:48]: Okay sure, check out my suggestion ^

**Comment by mbruin-NR** [comment]
Thank you @ally-sassman for all your great suggestions, they were a big help!

**Comment by ally-sassman** [comment]
@mbruin-NR You're welcome, and good work on this! Can I merge it now, or do you need to make any other changes?

**Review by ghost — DISMISSED** [review]

**Comment by mbruin-NR** [comment]
I'm going to get another member on my teams quick approval and then me can merge. Thanks again!

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 10]
```diff
@@ -18,17 +18,33 @@ redirects:
...
+ For information about the latest version, refer to the [release notes](/docs/releases/ios).
 
-You must be an account Admin to install, configure, and upgrade the iOS agent. For information about the latest version, refer to the [release notes](/docs/releases/ios).
+## Update with Swift Package Manager [#SPM]
 
-## Replace your iOS framework [#upgrading]
+To update the New Relic iOS agent using Swift Package Manager:
 
-First, you'll need to replace the earlier version of your iOS agent framework:
+1. Navigate to the **NewRelic** agent in the Package Dependencies section of your xCode project navigator.
```
Comment (smalsam-newr) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:27]:
```suggestion
1. Navigate to the **NewRelic** agent in the Package Dependencies section of your Xcode project navigator.
```

**Review by smalsam-newr — COMMENTED** [review]
Fix spelling of Xcode

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx [hunk 11]
```diff
@@ -18,17 +18,36 @@ redirects:
...
+2. Right-click on the **NewRelic** agent.
+3. Select **Update Package**.
 
-1. From the Project Navigator (**CMD 1**) in Xcode, search for **NewRelic.xcframework**.
-2. Right-click or control-click **NewRelic.xcframework**, and select **Show in Finder**.
-3. Drag **NewRelic.xcframework** to the trash.
-4. Verify that the Xcode project highlights the reference to **NewRelic.xcframework** in red.
-5. Right-click or control-click **NewRelic.xcframework**, and select **Delete** to remove the obsolete reference from the project.
+<Callout variant="important">
+Always check the [release notes](/docs/releases/ios) to make sure you are getting the most up to date agent versions. Depending on how your Swift Package Manager rules are set up you may need to change the `Up to Next Major` or Branch settings to include the new version.
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/upgrade-new-relic-mobiles-ios-sdk.mdx:32]:
```suggestion
Always check the [release notes](/docs/releases/ios) to make sure you're getting the most up-to-date agent versions. Depending on how your Swift Package Manager rules are set up, you may need to change the `Up to Next Major` or Branch settings to include the new version.
```
@mbruin-NR "Branch" needs to either be lowercased or code formatted.

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
See my suggestion and comment above.

**Review by ghost — APPROVED** [review]

**Comment by ally-sassman** [comment]
@mbruin-NR Can I merge this now?

**Comment by mbruin-NR** [comment]
This can be merged now. Thank you for your help!

**Pull request merged**